examples/tests/whiteboard.py

```diff
@@ -19,7 +18,6 @@ class WhiteBoard(Test):
     """
 
     def test(self):
-        os.chdir(self.datadir)    # Just to stress out Avocado
```

Comment:
This is unrelated change and it's here to stress Avocado. I think it should be kept but if you believe it does not belong to example tests send this as a separate commit (or PR to speed up the acceptance)